src/components/notifications/AddNotification.tsx

```diff
@@ -55,20 +73,22 @@ interface AddNotificationState {
     pipelineList: PipelineType[];
     filterInput: string;
     sesConfigId: number;
+    options:Options;
+    isApplistLoading:boolean
 }
 
 export class AddNotification extends Component<AddNotificationsProps, AddNotificationState> {
     filterOptionsMain = [
-        { value: "application", label: "application", type: "main" },
-        { value: "project", label: "project", type: "main" },
-        { value: "environment", label: "environment", type: "main" },
+        { value: 1, label: "application", type: "main" },
+        { value: 2, label: "project", type: "main" },
+        { value: 3, label: "environment", type: "main" },
     ];
     filterOptionsInner = [];
 
     constructor(props) {
         super(props);
         this.state = {
-            view: ViewType.LOADING,
+            view: ViewType.FORM,
```

Comment:
Why are we changing it from loading to form?

Reply:
done